lib/block/blockupload.js

```diff
@@ -181,7 +181,6 @@ class BlockUpload extends EventEmitter {
             }
         });
 
-        console.log("Block upload: block upload controller created");
```

Comment:
this log is after instantiating a class that does not depend on client input. Therefore this set up may only fail in a dev environment and this log is not needed in prod